frontend/src/icons/XMark.tsx

```diff
@@ -0,0 +1,9 @@
+const XMark = () => {
```

Comment:
다른 Icon 컴포넌트들처럼 ~Icon 으로 명명하는게 좋지 않을까요?
```suggestion
const XIcon = () => {
```

Reply:
그리고 툴팁에서는 닫기를 안쓰기로 했지만 최근검색어 삭제기능에 필요할 듯 해서 놔둬도 될 것 같습니다ㅎㅎ..

Reply:
넵 최근검색어 삭제에 쓰면 좋을 것 같아요 ㅎㅎ 이름은 수정하겠습니다~